commands/local_server_start.go

```diff
@@ -31,6 +31,7 @@ import (
 	"path/filepath"
 	"syscall"
 
+	"github.com/dunglas/frankenphp"
```

Comment:
Does it make sense to have here a go build tag? Frankenphp uses a lot of cgo which would break the release ci pipeline 🤔

Reply:
We could do that, but first I'll try to have to build a release pipeline supporting all current targets (they are supported by PHP, and we can use Docker to cross-compile).